src/services/errors.js

```diff
@@ -19,8 +19,16 @@ module.exports= {
 		name: 'TagAlreadyAssignedToPostError',
 		message: 'Tag is already assigned to post '
 	},
-	refreshTokenIsInvalidError : {
-		name : 'refreshTokenIsInvalidError',
+	RefreshTokenIsInvalidError : {
+		name : 'RefreshTokenIsInvalidError',
 		message : "Refresh token is invalid."
+	},
+	InvalidInviteError : {
+		name : "InvalidInviteError",
+		message: "Invite is invalid/expired"
+	},
+	InviteStatusActiveError : {
+		name : "InviteStatusActiveError",
+		message: "Invite status is active"
```

Comment:
frontend doesn't know much about 'status' being active or not. we should capture that invite is not valid, we can simply use the invalid invite one. that's fine

Reply:
yes please